teamA/lib/Views/dashboard.dart

```diff
@@ -0,0 +1,279 @@
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+final ColorScheme customColorScheme = ColorScheme(
+  primary: Color(0xFF6A5A99), // Purple color as primary
+  onPrimary: Color(0xFFFFFFFF), // White text on primary
+  primaryContainer: Color(0xFFEDE6FF), // Light lavender
+  onPrimaryContainer: Color(0xFF2D004A), // Dark purple
+  secondary: Color(0xFF6A5A99), // Using similar purple as secondary
+  onSecondary: Colors.white, // White on secondary
+  background: Color(0xFFF5F5F5), // Light grey background
+  onBackground: Colors.black, // Black text on background
+  surface: Color(0xFFFFFFFF), // White surface color
+  onSurface: Colors.black, // Black text on surface
+  error: Colors.red, // Error color red
+  onError: Colors.white, // White on error
+  brightness: Brightness.light, // Light mode
+);
```

Comment:
Color Scheme isn't really necessary unless you want a specific look.

Reply:
i needed this color scheme because the buttons uses color and shadows, but i removed it now.